Add meta descriptions to Lisk-SDK, closes #371 (#387)
LiskHQ/lisk-docs: modules/ROOT/pages/lisk-commander/user-guide/commands.adoc

= Lisk Commander Reference
Mona Bärenfänger <mona@lightcurve.io>

> :description: The Lisk Commander Command Reference lists all available commands of Lisk Commander as a technical reference. For each it comprises all available options, a usage example and an example response.

Reviewer: ```suggestion
:description: The Lisk Commander Command Reference lists each available command of Lisk Commander as a technical reference. For all commands this consists of all available options, a usage example and an example response.
```
